DEV-8053 Link to Federal Account profiles status of funds viz

afnazu opened the pull request:
**High level description:**

*Do not merge until dev-8052 is merged

Labels on federal account level are links to federal account pages

**JIRA Ticket:**
[DEV-8053](https://federal-spending-transparency.atlassian.net/browse/DEV-8053)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [n/a] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [n/a] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [n/a] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [n/a] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [n/a] Design review complete `if applicable`
- [n/a] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

brett-varney reviewed (changes requested):
This solution works very well for "normal" users, but I don't think is 508-compliant. The labels are navigable by keyboard, but they don't activate with Enter or Space. Further, there is no indication to screen readers that these are hyperlinks.

afnazu commented:
@brett-varney Thanks I've updated with enter/space functionality and added aria-labels for the links. Just tested with a screenreader and the links are being read as "Link to [federal account name]" at the subcomponent level and read as the label itself at the agency level

brett-varney commented:
Good solution for WCAG
